apps/fdsnws/test/test-station.py

```diff
@@ -31,6 +31,7 @@ def test(self):
             ('?format=text&startbefore=2019-07-01', TestStationBase.CT_TXT, [], False),
             ('?level=channel&includeavailability=true', TestStationBase.CT_XML, [(172, 198, 7, 0)], False),
             ('?format=sc3ml&network=AM&station=R0F05&location=00&channel=SHZ&latitude=52&longitude=13&maxradius=0.5&level=response&includeavailability=true', TestStationBase.CT_XML, [], True),
+            ('?network=DK&station=OVD', TestStationBase.CT_XML, [(172, 198)], False)
```

Comment:
The Created time stamp between character 172 and 198 is not only dynamic but also variable in length. The microsecond fraction part can contain 0-7 characters. Right now 1 out of 10 test runs fail. Use

```
('?network=DK&station=OVD', TestStationBase.CT_XML, [(172, 198, 7)], False)
```

Aside from this problem the PR works under Python 2 and 3 and is ready to be merged.